server/chat-commands/moderation.ts

```diff
@@ -926,9 +921,7 @@ export const commands: ChatCommands = {
 		this.addModAction(banMessage);
 
 		// Notify staff room when a user is banned outside of it.
-		if (room.roomid !== 'staff' && Rooms.get('staff')) {
-			Rooms.get('staff')?.addByUser(user, `<<${room.roomid}>> ${banMessage}`);
-		}
+		Rooms.get('staff')?.addByUser(user, `${room ? `<<${room.roomid}>>` : `<PMs: ${this.pmTarget}>`} ${banMessage}`);
```

Comment:
This code even has a comment describing what it's for.

Without the `room.roomid !== 'staff'` part, if someone bans a user in Staff, this would add two redundant messages in Staff:

```
Spammer was globally banned by Staff. (reason)
<<staff>> Spammer was globally banned by Staff. (reason)
```